oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndex.java

```diff
@@ -1677,6 +1708,28 @@ private List<Facet> getFacetsUncached(int numberOfFacets, String columnName) thr
                 indexNode.release();
             }
         }
+
+        private List<Facet> getFacetsUncached(Facets facets, int numberOfFacets, String columnName) throws IOException {
+            String facetFieldName = FulltextIndex.parseFacetField(columnName);
+            try {
+                ImmutableList.Builder<Facet> res = new ImmutableList.Builder<>();
+                FacetResult topChildren = facets.getTopChildren(numberOfFacets, facetFieldName);
+                if (topChildren == null) {
+                    return null;
+                }
+                for (LabelAndValue lav : topChildren.labelValues) {
+                    res.add(new Facet(
+                            lav.label, lav.value.intValue()
+                    ));
+                }
+                return res.build();
+            } catch (IllegalArgumentException iae) {
+                LOG.debug(iae.getMessage(), iae);
+                LOG.warn("facets for {} not yet indexed: " + iae, facetFieldName);
```

Comment:
this message mixes placeholders + interpolation. I would prefer to use just placeholders:
```suggestion
                LOG.warn("facets for {} not yet indexed: {}", facetFieldName, iae);
```